source/Graphics.cs

```diff
@@ -1,35 +1,51 @@
 ﻿// Copyright (C) 2022 Maxim Gumin, The MIT License (MIT)
 
 using System;
-using System.Drawing;
-using System.Drawing.Imaging;
+using System.Buffers.Binary;
 using System.Collections.Generic;
+using SixLabors.ImageSharp;
+using SixLabors.ImageSharp.PixelFormats;
 
 static class Graphics
 {
     public static (int[], int, int, int) LoadBitmap(string filename)
     {
         try
         {
-            Bitmap bitmap = new(filename);
-            int width = bitmap.Width, height = bitmap.Height;
-            var bits = bitmap.LockBits(new Rectangle(0, 0, width, height), ImageLockMode.ReadOnly, PixelFormat.Format32bppArgb);
-            int[] result = new int[bitmap.Width * bitmap.Height];
-            System.Runtime.InteropServices.Marshal.Copy(bits.Scan0, result, 0, result.Length);
-            bitmap.UnlockBits(bits);
-            bitmap.Dispose();
+            var image = Image.Load<Bgra32>(filename);
+            int width = image.Width, height = image.Height;
+            var result = new int[width * height];
+            for (var j = 0; j < height; j += 1)
+            {
+                for (var i = 0; i < width; i += 1)
+                {
+                    result[j * width + i] = unchecked((int) image[i, j].Bgra);
+                }
+            }
+
+            image.Dispose();
```

Comment:
You can use a `using` to avoid the explicit call to `Dispose`.
But most importantly, you can vectorize all this with the built-in API to copy the pixel buffer:
```suggestion
            using var image = Image.Load<Bgra32>(filename);

            int width = image.Width;
            int height = image.Height;
            int[] result = new int[width * height];

            image.CopyPixelDataTo(MemoryMarshal.Cast<int, Bgra32>(result));
```